### ISSUE TITLE
Use of uninitialized value in require at /usr/lib/x86_64-linux-gnu/perl5/5.34/Encode.pm line 70.

### ISSUE THREAD

**Issue by RightCake1** [sanger-pathogens/Roary#577]
"![Screenshot from 2022-08-01 11-42-35](https://user-images.githubusercontent.com/72471432/182079967-4ad0d767-5528-4c30-bc14-10cbfdf364ea.png)

I'm fairly new to this and I'm getting this when i type in " roary -e --mafft -p 8 -f ROARY_dir **.gff  
what should i do?

**Comment by Barrington-wustl** [comment]
i am having the same issue did you find a solution?

**Comment by RightCake1** [comment]
Yes, i purged the file and reinstalled it. It worked fine after that.
Hope that helps you as well.

**Comment by Barrington-wustl** [comment]
did you purge it by `sudo apt remove roary` because that isn't working for me

**Comment by RightCake1** [comment]
I had some dependancies that were half installed and causing the issues. If
i remember correctly, it was probably something libs. so i just purged that
and tried installing it fresh. and then install roary.

**Comment by lskatz** [comment]
@RightCake1 what was your command to purge and then reinstall?

**Comment by RightCake1** [comment]
> @RightCake1 what was your command to purge and then reinstall?

Hellow brother. I believe it was something like " sudo apt purge program_name ". I honestly forgot which app it was. I can just recall that it was  like lib x86 something that wasnt properly installed due to network issues at that time. 
I hope that helps!

**Comment by zxk514** [comment]
I also got a same issue. Someone can give me useful advice to fix the question?

**Comment by 451106689** [comment]
I found that this error did not affect the normal operation.
